### PR TITLE
Add Interface Null Norm PyBinding

### PR DESCRIPTION
**Pull request by estherroselopez**
## Proposed changes

Added the Python Binding for Interface Null Normal.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by nilsvu — DISMISSED** [review]

**Comment by nilsvu** [comment]
Please rebase on the latest `develop` branch to resolve the conflicts. Note that we switched Python formatting recently, so look at the upgrade instructions in #3730 if that's an issue (or ask me).

**Review by nilsvu — DISMISSED** [review]

File: src/NumericalAlgorithms/Spectral/Spectral.hpp [hunk 1]
```diff
@@ -160,7 +160,7 @@ constexpr size_t minimum_number_of_points =
  */
 template <Basis basis>
 constexpr size_t maximum_number_of_points =
-    basis == Basis::FiniteDifference ? 24 : 12;
```
Comment (nilsvu) [inline src/NumericalAlgorithms/Spectral/Spectral.hpp]: Please remove this change (we should probably do it, but in another PR)

**Review by nilsvu — CHANGES_REQUESTED** [review]
Please squash the two commits together.

**Review by nilsvu — DISMISSED** [review]

**Review by nilsdeppe — COMMENTED** [review]
@nilsvu could you take a look at this please?

**Review by nilsvu — APPROVED** [review]

**Comment by nilsvu** [comment]
Looks good! Thanks @estherroselopez. Since this is your first contribution we invite you to add your name to the author list of the code in the `Metadata.yaml` file. Just add an entry for yourself in `Authors.Contributors` and open a pull request with the change. Once the pull request is merged, your name will appear on the [SpECTRE DOI on Zenodo](https://doi.org/10.5281/zenodo.4290404) with the next public release.

**Pull request merged**